staging/src/k8s.io/apiserver/pkg/server/httplog/httplog.go

```diff
@@ -157,22 +157,23 @@ func (rl *respLogger) Addf(format string, data ...interface{}) {
 func (rl *respLogger) Log() {
 	latency := time.Since(rl.startTime)
 	if klog.V(3).Enabled() {
-		if !rl.hijacked {
-			klog.InfoDepth(1, fmt.Sprintf("verb=%q URI=%q latency=%v resp=%v UserAgent=%q srcIP=%q: %v%v",
-				rl.req.Method, rl.req.RequestURI,
-				latency, rl.status,
-				rl.req.UserAgent(), rl.req.RemoteAddr,
-				rl.statusStack, rl.addedInfo,
-			))
-		} else {
-			klog.InfoDepth(1, fmt.Sprintf("verb=%q URI=%q latency=%v UserAgent=%q srcIP=%q: hijacked",
-				rl.req.Method, rl.req.RequestURI,
-				latency, rl.req.UserAgent(), rl.req.RemoteAddr,
-			))
-		}
+		klog.V(3).InfoS("Recieved HTTP Request", rl.LogArgs(latency)...)
 	}
 }
 
+
+func (rl *respLogger) LogArgs(latency time.Duration) []interface{} {
+		return []interface{}{
+				"Hijacked", rl.hijacked,
```

Comment:
please add hijacked key only if rl.hijacked is true
This log is generates most log volume in k8s. We shouldn't add this key unnecessary.

Reply:
Okay, I will put a if condition there.